lib/request_started_on_middleware.rb

```diff
@@ -14,13 +14,13 @@ def call(env)
     complete_request
   end
 
-  def start_request(path, started_on)
-    Thread.current[:current_request] = path
+  def start_request(path_info, started_on)
+    Thread.current[:current_request_path_info]  = path_info
```

Comment:
Should we toss in a `miq_...` at the beginning just for good measure?

Reply:
This is epic :trollface:

💯

Reply:
I think this is good. I actually had `current_request_path` initially and LJ suggested this longer name.

Reply:
> This is epic :trollface:

@jrafanie For the record, this was actually not the intent, and I meant it in all seriousness.  This is (arguably) still a variable name that a different library could bring in, but it is highly unlikely that someone is going to add `miq_*` to their Thread variable names (besides us).

Reply:
> > This is epic :trollface:
>
> @jrafanie For the record, this was actually not the intent, and I meant it in all seriousness. This is (arguably) still a variable name that a different library could bring in, but it is highly unlikely that someone is going to add `miq_*` to their Thread variable names (besides us).

That's made it great.  I couldn't tell if you were serious or joking.  🤣

Reply:
Yeah, I agree with brevity especially since `current_request_path_info` is quite specific.  I guess you could call it `current_request_env_path_info` to make it more specific but then you could keep doing that until the name losing any meaning.